docs/howtos/prometheus.md

```diff
@@ -255,7 +255,7 @@ spec:
               protocol: TCP
           env:
             - name: GF_SERVER_ROOT_URL
-              value: {{AMBASSADOR_IP}}/grafana
+              value: https://{{AMBASSADOR_IP}}/grafana
```

Comment:
I personally thing this should be `{{SCHEME}}://{{AMBASSADOR_HOST}}`/grafana since you would need to use the hostname of your Ambassador if you have one

That or `{{AMBASSADOR_ORIGIN}}` since an origin is a `scheme+hostname`. We would need to add an example of what this means since a lot of people do not know what the difference between a `host` and origin are